src/components/Banner.jsx

```diff
@@ -5,9 +5,8 @@ const Banner = () => {
   return (
     <div className="beta-top-banner">
       <h2>
-        Thanks for participating in our beta test.
-        Please click on the "Sign in" button at top right
-        to login or create an account.
+        We’re currently implementing some changes to the project based on
```

Comment:
_Shakes fists angrily at slanted apostrophe_ Nyyyeeeaaarrrgh!

OK, this doesn't affect this PR in any way, but I need to complain about this. That curly apostrophe was an enemy of mine since the IE7/IE8 days because it's not a proper ASCII-safe `'` single quote used in coding, but a display-only character `’` formatted by word processors. It caused me hella problems back when browsers couldn't render it properly, but it's less of a problem in an era where _emojis_ are considered valid characters. 🤷‍♂️ 

In case you can't tell the difference between the two:

`It's Shaun's pet peeve`
vs
`It’s Shaun’s pet peeve`